src/vmstats_server.erl

```diff
@@ -68,6 +68,13 @@ handle_info({timeout, R, ?TIMER_MSG}, S = #state{sink=Sink, key=K, key_separator
     Sink:collect(gauge, [K,"port_count"], erlang:system_info(port_count)),
     Sink:collect(gauge, [K,"port_limit"], erlang:system_info(port_limit)),
 
+    %% Atom count, working only on Erlang 20+
+    try
+        Sink:collect(gauge, [K, "atom_count"], erlang:system_info(atom_count))
+    catch
+        _:_ -> ok
```

Comment:
We should only catch `error:undef` here to avoid eating up other weird errors silently.

Reply:
Fixed. The error to catch is `badarg` since the fun exists for a longer time.